lib/cocoapods/installer/analyzer.rb

```diff
@@ -1119,7 +1119,7 @@ def generate_specifications(resolver_specs_by_target)
       def generate_sandbox_state(specifications)
         sandbox_state = nil
         UI.section 'Comparing resolved specification to the sandbox manifest' do
-          sandbox_analyzer = SandboxAnalyzer.new(sandbox, specifications, update_mode?)
+          sandbox_analyzer = SandboxAnalyzer.new(sandbox, @podfile, specifications, update_mode?)
```

Comment:
just use `podfile` here since there is an `attr_reader` defined at the top.

Reply:
thanks for your review